.github/workflows/package_pixi.yaml

```diff
@@ -0,0 +1,54 @@
+name: conda packaging and deployment
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [qa, main]
+    tags: ['v*']
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: prefix-dev/setup-pixi@v0.8.3
+        with:
+          pixi-version: v0.41.4
```

Comment:
Do you really need to specify a version of pixi? I would have expected that the reusable action would have selected an appropriate version. At least, it would be good if we could not specify the patch-level

Reply:
This is following pixi's official documentation (https://pixi.sh/latest/advanced/github_actions/). The issue with pixi is that it still consider itself in a rapid development stage and the API might change between minor versions. So the example is recommending us to pin the number. Maybe in the future we can remove it after pixi gains some confidence in itself.